services/movie_session.py

```diff
@@ -0,0 +1,50 @@
+from db.models import MovieSession, Movie, CinemaHall
+from typing import List, Optional
+import datetime
+
+
+def create_movie_session(
+        movie_show_time: datetime.datetime,
+        movie_id: int,
+        cinema_hall_id: int
+) -> MovieSession:
+    movie = Movie.objects.get(id=movie_id)
+    cinema_hall = CinemaHall.objects.get(id=cinema_hall_id)
+    return MovieSession.objects.create(
+        show_time=movie_show_time,
+        movie=movie,
+        cinema_hall=cinema_hall
```

Comment:
You could just set `cinema_hall_id`= `cinema_hall_id` . Same with movie . Instead of set whole instance of movie or cinema_hall